muskathlon/models/event_compassion.py

```diff
@@ -85,3 +86,14 @@ def get_muskathlon_participants(self, event_id):
                 'country': participant.partner_id.country_id.name
             })
         return ret
+
+    @api.multi
+    def _cron_delete_medical_surveys(self):
+        for event in self.env['crm.event.compassion'].search([
+            ('end_date', '<',
+             fields.Datetime.to_string(datetime.now() - timedelta(days=31)))]):
+            for registration in event.registration_ids:
+                if len(registration.medical_survey_id) > 0:
```

Comment:
No need to add this test. unlink() method will simply do nothing if the relation is empty (will return an empty recordset and unlink will do nothing).